### PR TITLE
[types-2.0] D3 Upgrade to version 4.4

### PR DESCRIPTION
**Pull request by tomwanzek**
Please fill in this template.

- [x] Prefer to make your PR against the `types-2.0` branch.
- [x] Test the change in your own code.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).
- [x] Run `npm run lint -- package-name` if a `tslint.json` is present.

If adding a new definition:
- [ ] The package does not provide its own types, and you can not add them.
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Run `tsc` without errors.
- [ ] Include the required [files](https://github.com/DefinitelyTyped/DefinitelyTyped#create-a-new-package) and header. Base these on the README, *not* on an existing project.

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://github.com/d3/d3/blob/master/package.json>>
- [x] Increase the version number in the header if appropriate.

d3 [Chore]:
* Update dependencies for d3-geo to 1.4 and d3-zoom to 1.1 to reflect d3 Standard Bundle version 4.4
* Exempted _legacy_ definitions and tests from TSLint at file level.

### CODE REVIEW

**Comment by gustavderdrache** [comment]
:+1:

**Comment by tomwanzek** [comment]
@andy-ms when you have a second, this one is ready to go! Cheers, T

File: d3/d3-v3.5-tests.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-
+/* tslint:disable */
```
Comment (andy-ms) [inline d3/d3-v3.5-tests.ts]: This shouldn't be necessary in a test file.

**Comment by tomwanzek** [comment]
@andy-ms Done. And Travis is smiling already....

**Pull request merged**